mango-python/bdgenomics/mango/QC.py

```diff
@@ -19,6 +19,8 @@
 from collections import Counter, OrderedDict
```

Comment:
I think we should start breaking these off into separate files, so we have a different file for variants, reads, etc.